src/realm/table.cpp

```diff
@@ -1031,6 +1032,11 @@ void Table::do_erase_root_column(ColKey col_key)
     bump_storage_version();
 }
 
+Query Table::where(const DictionaryLinkValues& dictionary_of_links) const
```

Comment:
Why is this one still needed, can't we always use the ObjList version?

Reply:
There is no automatic cast from Dictionary to ObjList, so without this one, you would have to do that explicitly.